packages/styles/scss/components/quote/_quote.scss

```diff
@@ -22,7 +23,20 @@
     }
   }
 
-  .#{$prefix}--quote__container {
+  :host(#{$dds-prefix}-quote).#{$prefix}--quote__inverse {
```

Comment:
Instead of adding a CSS class that replaces `inverse` attribute, would you try simply making the quote work in dark theme? Please don't hesitate ping me offline if you have any questions. Thanks!